core/src/test/java/org/elasticsearch/action/search/MultiSearchRequestTests.java

```diff
@@ -119,10 +133,47 @@ public void testSimpleAdd4() throws Exception {
     }
 
     public void testResponseErrorToXContent() throws IOException {
-        MultiSearchResponse response = new MultiSearchResponse(new MultiSearchResponse.Item[]{new MultiSearchResponse.Item(null, new IllegalStateException("foobar")), new MultiSearchResponse.Item(null, new IllegalStateException("baaaaaazzzz"))});
-        XContentBuilder builder = XContentFactory.jsonBuilder();
+        MultiSearchResponse response = new MultiSearchResponse(
+                new MultiSearchResponse.Item[]{
+                    new MultiSearchResponse.Item(null, new IllegalStateException("foobar")),
+                    new MultiSearchResponse.Item(null, new IllegalStateException("baaaaaazzzz"))
+        });
+
+        XContentBuilder builder = XContentFactory.jsonBuilder().prettyPrint();
+        builder.startObject();
         response.toXContent(builder, ToXContent.EMPTY_PARAMS);
-        assertEquals("\"responses\"[{\"error\":{\"root_cause\":[{\"type\":\"illegal_state_exception\",\"reason\":\"foobar\"}],\"type\":\"illegal_state_exception\",\"reason\":\"foobar\"},\"status\":500},{\"error\":{\"root_cause\":[{\"type\":\"illegal_state_exception\",\"reason\":\"baaaaaazzzz\"}],\"type\":\"illegal_state_exception\",\"reason\":\"baaaaaazzzz\"},\"status\":500}]",
+        builder.endObject();
+
+        assertEquals("{\n" +
```

Comment:
pretty printing is difficult, not sure if it works on windows with trailing whiltespaces / new lines etc?